src/style.css

```diff
@@ -0,0 +1,123 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+}
+
+body {
+  margin: 0;
+  line-height: 14px;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 406px;
+  display: flex;
+  flex-direction: column;
+  background: rgb(255, 255, 255);
+  border: 1px solid rgb(243, 243, 243);
+  border-radius: 5px;
+}
```

Comment:
you can add padding for card one time here , and dont`t repeat it another elements.

For example: ` padding: 32px 16px 16px;` :)

Reply:
where i didn't use that ? )